scraper/src/youtube2zim/schemas.py

```diff
@@ -100,3 +100,10 @@ class Channel(CamelModel):
     joined_date: str
     collection_type: str
     main_playlist: str | None = None
+
+
+class Config(CamelModel):
+    """Class to serialize configuration data for the ZIM."""
```

Comment:
"for the ZIM." => "for the ZIM UI." ?

Reply:
Changed in [35c4e76](https://github.com/openzim/youtube/pull/263/commits/35c4e767b329aacee1f51d67e8f3b15196163f97)